adr/0021-rules-for-entity-mapping-in-service-catalog.md

```diff
@@ -0,0 +1,175 @@
+# Rules for entity mapping in Service Catalog
+
+* Status: Proposed
+* Deciders: mimotej, SamoKopecky, tumido
+* Date: 2022-07-26
+
+## Context and Problem Statement
+
+Backstage offers multiple resource types which can be used to represent entities and their relations in service catalog. We need to have a set of rules present in Operate First which will provide context and guidance on how entities are mapped to Backstage resources.
```

Comment:
```suggestion
Backstage offers multiple resource types which can be used to represent entities and their relations in a service catalog. We need to have a set of rules present in Operate First that will provide context and guidance on how entities are mapped to Backstage resources.
```


I think here should be either `a` or `the` also I think in this context there should be `that` instead of `which`